thoth/storages/graph/postgres.py

```diff
@@ -371,6 +371,14 @@ def map_os_name(os_name: Optional[str]) -> Optional[str]:
             return "rhel"
 
         return os_name
+    
+    @staticmethod
+    def map_python_index_url(index_url: Optional[str]) -> Optional[str]:
```

Comment:
Can this be renamed to `normalize_python_index_url` so it matches other normalize_* methods?

Reply:
Thanks!